src/store.js

```diff
@@ -1 +1,22 @@
 
+
```

Comment:
in jest, config ignore store and other .config files 
which are bringing  the coverage down

Reply:
@Kachulio1  I have ignored the files that are bringing the coverage down.